src/test/rustdoc/scrape-examples-wrong-options.rs

```diff
@@ -0,0 +1,2 @@
+// should-fail
+// compile-flags: --scrape-examples-target-crate foobar
```

Comment:
This doesn't actually test anything, I don't think // should-fail has an effect for rustdoc tests. Can you move this to rustdoc-ui and use --error-pattern instead so it tests the output? https://rustc-dev-guide.rust-lang.org/tests/adding.html?highlight=Error-pattern#when-error-line-cannot-be-specified